src/native/other/translateText.ts

```diff
@@ -0,0 +1,42 @@
+import { ArgType, NativeFunction } from "../../structures";
+import translate from "@iamtraction/google-translate";
+
+export default new NativeFunction({
+    name: "$translateText",
+    version: "1.1.3",
+    description: "translates text into another language.",
+    brackets: true,
+    args: [
+        {
+            name: "text",
+            description: "text to translate",
+            type: ArgType.String,
+            required: true,
+            rest: false
+        },
+        {
+            name: "to",
+            description: "The Language to translate to",
+            type: ArgType.String,
+            required: true,
+            rest: false
+        },
+        {
+            name: "from",
+            description: "The Language to translate from.",
+            type: ArgType.String,
+            required: false,
+            rest: false
+        }
+    ],
+    unwrap: true,
+    async execute(ctx, [text, toLang, fromLang]) {
+        
+        try {
+        const res = await translate(text, { from: fromLang ?? "auto", to: toLang })
+        return this.success(res.text);
+    } catch (e) {
+        return this.customError(`Translation failed`);
+    }
+    }
+});
```

Comment:
```suggestion
    async execute(ctx, [text, toLang, fromLang, json]) {
        const res = await translate(text, { to: toLang, from: fromLang || "auto" }).catch(ctx.noop)
        if (json) return this.successJSON(res)
        return this.success(res?.text)
    }
})
```
Multiple issues here:
1. Spacing.
2. Using `??` will allow an empty string as input if the `from` argument was provided without a value. To prevent this use `||`.
3. We do not send out custom errors if an error occurred, instead the error will be logged directly to the console (read next point) and the function simply won't return anything.
4. Do not use a `try` and `catch` block here, instead use `.catch(ctx.noop)` to log possible errors to the console.
5. Please do not use `;` unless it's mandatory. Also applies outside of the code.

As you may have noticed, I've added a way to return the whole response as JSON object. This allows accessing more information regarding the translation :)